.github/pull_request_template.md

```diff
@@ -1,10 +1,38 @@
-This PR implements the following **changes:**
+## summary
 
-* insert your summary of your request here, add more bullets as needed
+Basic description of work done.
 
+[Link to Preview]()
 
-**URL / Link to page**
+## steps to recreate
 
-<!-- Link to the page that will be changed -->
-<!-- Delete this section if not needed -->
+## solution
 
+### markdown changes
+### features/enhancements/bugs
+
+- [ ] item 1
+- [ ] item 2
```

Comment:
Not sure what L11-15 are supposed to represent. Let's clarify or remove for simplicity. We can always comeback in the future and update.